# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: toolkit/components/find/nsFind.cpp
# Commit: c1e64e8e4665
# Full Hash: c1e64e8e466506d1d9a36d747df25527bd545b10
# Author: Martin Robinson <mrobinson@igalia.com>
# Date: 2022-12-01 16:18:29
# Regressor Bug: 1791759
# File Overlap Count: 1
# Description:
#   Bug 1791759 - Add support for `content-visibility: auto` r=emilio
#   
#   This change adds support for `content-visibilty: auto` as well as
#   showing and hiding content based on the relevancy of the content as
#   defined in the specification. Changes to relevancy are handled by
# ==============================================================================

diff -r 6d4d56f57b7b -r c1e64e8e4665 toolkit/components/find/nsFind.cpp
--- a/toolkit/components/find/nsFind.cpp	Thu Dec 01 01:23:08 2022 +0200
+++ b/toolkit/components/find/nsFind.cpp	Wed Nov 30 18:00:27 2022 +0000
@@ -143,8 +143,9 @@
     return true;
   }
 
-  if (frame->HidesContent() ||
-      frame->IsHiddenByContentVisibilityOnAnyAncestor()) {
+  if (frame->HidesContent(nsIFrame::IncludeContentVisibility::Hidden) ||
+      frame->IsHiddenByContentVisibilityOnAnyAncestor(
+          nsIFrame::IncludeContentVisibility::Hidden)) {
     return false;
   }
 
